casanovo/denovo/dataloaders.py

```diff
@@ -2,19 +2,24 @@
 
 import functools
 import os
+import logging
 from typing import List, Optional, Tuple
 
+from depthcharge.data import AnnotatedSpectrumIndex
```

Comment:
**todo:** `from ... import ...` should come below `import ...`.

Reply:
Moved and fixed. Thanks for the catches, not sure how I missed that first one...